GNUmakefile

```
@@ -102,8 +102,7 @@ generate: abigen codecgen mockery ## Execute all go:generate commands.
 .PHONY: testscripts
 testscripts: chainlink ## Install and run testscript against testdata/scripts/* files.
 	go install github.com/rogpeppe/go-internal/cmd/testscript@latest
-	find testdata/scripts -type d | xargs -I % \
-	sh -c 'ls %/*.txtar > /dev/null 2>&1 || return 0 && PATH=$(CURDIR):$(PATH) testscript -e CL_DEV=true -e COMMIT_SHA=$(COMMIT_SHA) -e VERSION=$(VERSION) $(TS_FLAGS) %/*.txtar'
+	PATH=$(CURDIR):$(PATH) testscript -e CL_DEV=true -e COMMIT_SHA=$(COMMIT_SHA) -e VERSION=$(VERSION) $(TS_FLAGS) $$(go run ./testdata/scripts/ls-txtar-files)
```

Can we preserve the old behavior of invoking once per directory instead? It keeps things aligned with the internal `main_test.go`.
```suggestion
	./ls-txtar-dirs | xargs -I % \
	PATH=$(CURDIR):$(PATH) testscript -e CL_DEV=true -e COMMIT_SHA=$(COMMIT_SHA) -e VERSION=$(VERSION) $(TS_FLAGS) %/*.txtar
```